platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/IconManager.java

```diff
@@ -40,56 +41,60 @@
   }
 
   Icon loadIconForMarker(Marker marker) {
```

Comment:
This method is really long. What about extracting some blocks of code into `private` methods?